api/utils.go

```diff
@@ -64,10 +64,14 @@ func newClientAndScheme(tlsConfig *tls.Config) (*http.Client, string) {
 func getContainerFromVars(c *context, vars map[string]string) (string, *cluster.Container, error) {
 	if name, ok := vars["name"]; ok {
 		if container := c.cluster.Container(name); container != nil {
+			if !container.Engine.IsHealthy() {
+				return name, nil, fmt.Errorf("Container running on unhealthy node: %s", name)
```

Comment:
nit: It doesn't look right to put container name after `unhealthy node:`. How about showing container name and node name? User would be curious on the failing node.